hil/api.py

```diff
@@ -1131,27 +1131,31 @@ def port_revert(switch, port):
 
     db.session.add(action)
     db.session.commit()
-    return json.dumps({'status_id': unique_id})
+    return json.dumps({'status_id': unique_id}), 202
```

Comment:
202 doesn't make sense here; it indicates that processing is incomplete, but the request "get the status of ..." is complete.

Reply:
> it indicates that processing is incomplete

But that is true here (`revert_port`), isn't it? The other network calls `node_{connect, detach}_network` also return 202.